src/application-manager.coffee

```diff
@@ -693,17 +716,32 @@ module.exports = class ApplicationManager extends EventEmitter
 					Promise.map appsForDB, (app) =>
 						@db.upsertModel('app', app, { appId: app.appId }, trx)
 				.then (appsForDB) ->
-					trx('app').where({ source }).whereNotIn('appId', _.map(appsForDB, 'appId')).del()
+					trx('app').where({ source }).whereNotIn('appId',
+						# Use apps here, rather than filteredApps, to
+						# avoid removing a release whose subsequent
```

Comment:
`whose subsequent` what? Or `which subsequently`? Bit confused. (And 'which' not 'whose').